wfe/web-front-end.go

```diff
@@ -345,6 +355,26 @@ func (wfe *WebFrontEndImpl) verifyPOST(request *http.Request, regCheck bool) ([]
 		reg = core.Registration{}
 	}
 
+	// Check that the "resource" field is present and has the correct value
+	var parsedRequest map[string]interface{}
```

Comment:
please make a little struct for this. It makes it easier to document and avoids some of the noise we have below with the conversions